src/features/swagger/components/Navigation/Navigation.tsx

```
@@ -0,0 +1,31 @@
+import styles from './styles';
+
+function Navigation(): React.ReactElement {
+  const { t } = useTranslation();
+
+  const navigateHome: () => void = () => {
+    window.location.assign('/');
+  };
+
+  return (
+    <Box sx={styles.navigationWrapper} onClick={navigateHome}>
+      <Stack direction="row" role="navigation" sx={styles.navigationButton}>
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<nav>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)